activity_hub.php

```diff
@@ -400,13 +400,12 @@ function summarize_stage($stage, $desired_states, $show_filtered_projects = fals
         echo "<td class='pages-completed'>{$round_stats->curr_day_actual}</td>";
         echo "<td><div class='progressbar $progress_bar_class' style='width: $progress_bar_width%;'>&nbsp;</div><p style='clear: both; margin: 0;'>$percent_complete%</p></td>";
     } else {
-        // IE6 & 7 do not display the top and left borders of a cell if
-        // border-collapse=collapse and the border=1px. The following
-        // tweak is to ensure that if we show the filter that the filter
-        // cell has a border all the way around it.
+        // This 'if' is not currently really needed, as far as I can
+        // see. If it's removed, the only change is that the same nocell
+        // cells that are added to fill out the PP and PPV number rows
+        // will be added to the SR row, and once we have a number in the
+        // SR row, we'll want the filter to show there, too.
         if ($show_filtered_projects) {
```

Comment:
My test user does not have 20 pages proofread so https://github.com/DistributedProofreaders/dproofreaders/blob/master/activity_hub.php#L118 prevents me from testing this easily. However I think it is best to just remove the if and the comment to simplify logic.

Reply:
With the `if` in place, and adding some color: 
<img width="633" alt="image" src="https://user-images.githubusercontent.com/36800239/118372209-13546b00-b576-11eb-9527-54a7ebd4775c.png">

With the `if` commented out: 
<img width="612" alt="image" src="https://user-images.githubusercontent.com/36800239/118372268-6cbc9a00-b576-11eb-9095-7c3a3ace0272.png">

Reply:
What users who have filters turned on see (verified that this occurs for both light and dark themes:
<img width="560" alt="image" src="https://user-images.githubusercontent.com/36800239/118373324-dc815380-b57b-11eb-8b3a-a1cdb1d49bf4.png">

Reply:
Agreed we should remove the comment and the `if`.